docs/api/covidcast_changelog.md

```diff
@@ -1,4 +1,4 @@
----
+        ---
```

Comment:
This change is probably unintentional and should be un-done, since I think it'll mess up the Markdown formatting

Reply:
Oops sorry about that, will do.